src/sentry/integrations/slack/endpoints/event.py

```diff
@@ -69,6 +72,46 @@ def _get_access_token(self, integration: Integration) -> Any:
     def on_url_verification(self, request: Request, data: Mapping[str, str]) -> Response:
         return self.respond({"challenge": data["challenge"]})
 
+    def prompt_link(
+        self,
+        data: Mapping[str, Any],
+        slack_request: SlackRequest,
+        integration: Integration,
+    ):
+        # This will break if multiple Sentry orgs are added
```

Comment:
Could you elaborate on `break`?

Is it that we will fail to link identities if the organization the user expects isn't the first one?

If so we might want to follow up in a second PR to handle that somehow, and for now include that in the messaging somehow

Reply:
Let's say there are 2 sentry orgs installed in a slack workspace and some users are only a part of one of the org, this piece of code just picks the first org to build the auth url. If the user isn't a part of the first sentry org, they won't be able to link their slack identity through this link. This is also the current behaviour of the `/link` command I think